README.md

```diff
@@ -80,6 +80,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [aho_corasick](https://github.com/chenkovsky/aho_corasick) - AhoCorasick algorithm
  * [bisect](https://github.com/spider-gazelle/bisect) - Inserting values into a sorted array
  * [bitfields](https://github.com/elorest/bitfields) - Pure Crystal implementation of BitFields. Handles encoding/decoding of bytes.
+ * [crie](https://github.com/c910335/crie) - Compile-time Trie
```

Comment:
Is the right word `Tree` here?

```suggestion
 * [crie](https://github.com/c910335/crie) - Compile-time Tree
```

Reply:
It's [Trie](https://en.wikipedia.org/wiki/Trie).